### PR TITLE
COMMERCE-12880 Create Sample CX shipping method

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-shipping-engine/src/main/java/com/liferay/commerce/shipping/engine/internal/taglib/FunctionCommerceShippingMethodConfigurationScreenNavigationEntry.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+					"shippingMethodTypeSettings",
+					functionCommerceShippingEngine.
+						getTypeSettingsUnicodeProperties());
+			}
+			else {
+				httpServletRequest.setAttribute(
+					"isDefaultValue", Boolean.FALSE);
+				httpServletRequest.setAttribute(
+					"shippingMethodTypeSettings",
+					typeSettingsUnicodeProperties);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-shipping-engine/src/main/java/com/liferay/commerce/shipping/engine/internal/taglib/FunctionCommerceShippingMethodConfigurationScreenNavigationEntry.java:134]: @luca-pellizzon request attributes need to be `*WebKeys*`, https://github.com/brianchandotcom/liferay-portal/pull/144387#issuecomment-1827651028

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:stop

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:stop

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
Failures are not related to the PR. This can be force-forwarded. @victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**